tutor/routes/home.py

```diff
@@ -14,4 +14,12 @@ def register_route():
 
 @app.route("/login", methods=['GET', 'POST'])
 def login_route():
-    return login.login()
\ No newline at end of file
+    return login.login()
+
```

Comment:
Similar to the comment above, arn't routs that are relates to courses supposed to be in a separate files such as `courses.py`?